cmd/cloud-controller-manager/main.go

```diff
@@ -85,17 +86,20 @@ func cloudInitializer(config *cloudcontrollerconfig.CompletedConfig) cloudprovid
 	// initialize cloud provider with the cloud provider name and config file provided
 	cloud, err := cloudprovider.InitCloudProvider(cloudConfig.Name, cloudConfig.CloudConfigFile)
 	if err != nil {
-		klog.Fatalf("Cloud provider could not be initialized: %v", err)
+		klog.ErrorS(err, "Cloud provider could not be initialized")
+		os.Exit(1)
 	}
 	if cloud == nil {
-		klog.Fatalf("Cloud provider is nil")
+		klog.ErrorS(nil, "Cloud provider is nil")
+		os.Exit(1)
 	}
 
 	if !cloud.HasClusterID() {
 		if config.ComponentConfig.KubeCloudShared.AllowUntaggedCloud {
-			klog.Warning("detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
+			klog.InfoS("Detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
```

Comment:
```suggestion
			klog.InfoS("detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
```